eos-migrate-image-defaults.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Migrate eos-image-defaults to /var/lib
+ConditionPathIsDirectory=/var/eos-image-defaults
+ConditionPathIsSymbolicLink=!/var/eos-image-defaults
```

Comment:
Hmm - I don't think that's better necessarily? These conditions are pretty specifically-focused to make sure this task runs in the exact condition it needs to, and not otherwise. I think you only need ConditionNeedsUpdate if you can't write more specific conditions and need an external reference to ensure your upgrade job gets run. I don't mind either way, but if we did it using ConditionNeedsUpdate I'd remove the other tasks (and we'd need Before=systemd-updates-done or whatever it's called).